daprdocs/content/en/reference/cli/dapr-annotate.md

```diff
@@ -0,0 +1,88 @@
+---
+type: docs
+title: "annotate CLI command reference"
+linkTitle: "annotate"
+description: "Add dapr annotatations to a Kubernetes configuration"
+---
+
+### Description
+
+Add dapr annotations to a Kubernetes configuration.
```

Comment:
I presume arguments here reflects this list https://v1-8.docs.dapr.io/reference/arguments-annotations-overview/ 
If so has this list been checked since I could not find dapr.io/sidecar-image as a value.

Reply:
If the above is true (the list) then I suggest the following update. 

```suggestion
Add Dapr annotations to a Kubernetes configuration. This enables you to change the Dapr annotations on a deployment other than those specified in the deployment file. See [Kubernetes annotations]({{< ref arguments-annotations-overview >}}) for a full description of each annotation.
```

Reply:
@msfussell I believe that this command is not for all the annotations as it exists. I am also thinking that all annotations should not be supported via this command. I am planning on creating a proposal to change this command to follow a helm based `--set` based format for 1.9. I will make note of the annotations needed to allowed to changed there.